sites/main-site/src/content/blog/2024/maintainers-minutes-2024-11-29.mdx

```diff
@@ -22,80 +23,109 @@ by providing brief summaries of the monthly team meetings.
 
 In the November meeting we discussed (amongst others) the following topics:
 
--   [Stricter SemVer versioning](stricter-semver-versioning)
--   [Push for BAM to CRAM?](push-for-bam-to-cram)
--   [Removal of `workflowCitation()`](removal-of-workflowcitation)
--   [Recent CI approach changes](recent-ci-approach-changes)
--   [Modules piling up](modules-piling-up)
+-   [Stricter SemVer versioning](#Stricter SemVer versioning)
+-   [Push for BAM to CRAM?](#Push for BAM to CRAM?)
+-   [Removal of `workflowCitation()`](#Removal of `workflowCitation()`)
+-   [Recent CI approach changes](#Recent CI approach changes)
+-   [Modules piling up](#Modules piling up)
```

Comment:
Nope these were links with the heading ids (which are automatically generated), I don't think your changes are still linking correctly